conf/caddy-services/api.conf.example

```diff
@@ -0,0 +1,51 @@
+0.0.0.0:9999 {
```

Comment:
Can the services.unifiedapi_port in pf.conf be used?

Reply:
That file isn't templated like the other caddy files.

Its not impossible but it would be a bit of a pain to do for a very little upside